go/e2e/handle.go

```diff
@@ -0,0 +1,19 @@
+package function
+
+import (
+	"context"
+	"fmt"
+	"function/resp"
+	"net/http"
+	"os"
+)
+
+func Handle(ctx context.Context, res http.ResponseWriter, req *http.Request) {
+	res.Header().Add("Content-Type", "text/plain")
+	res.WriteHeader(200)
+
+	_, err := fmt.Fprintf(res, resp.Get("REMOTE_VALID")+"\n")
```

Comment:
```suggestion
	_, err := fmt.Fprintf(res, "REMOTE_VALID\n")
```
The resp package is not doing anything to accomplish the E2E's goal so far as I can tell?  Probably best to leave it out until it's needed for a specific purpose

Reply:
Here I the purpose of the package is to indirectly ensure func create will download properly folders and content on it (although I saw you put together a test for that along with file mode check). Test/ Build would fail in case that go file is not present.

Reply:
OK that's fine.  A little outside the scope of ensuring you can use a remote template, because internal and external templates use the same "write files" logic, but maybe it will catch a regression in the future!